server/services/employers.js

```diff
@@ -75,16 +75,24 @@ const getSitesWithCriteria = async (additionalCriteria, additionalCriteriaParams
         employer_sites.body -> 'healthAuthority' as "healthAuthority",
         employer_sites.body -> 'postalCode' as "postalCode",
         employer_sites.body -> 'allocation' as "allocation",
-        count(ps.id) :: INT as "hireCount"
+        (
+          SELECT
+              count(*)
+          FROM
+              participants_status ps
+              LEFT outer JOIN participants_status ps_da on ps.participant_id = ps_da.participant_id
+              AND ps_da.status = 'archived'
```

Comment:
Sorry, why archived?

Reply:
With the self join this is a non-straightforward way to get hired statuses where people aren't archived with the following line  `and ps_da.data ->> 'type' is null`

This a based off the other count for hired participants https://github.com/bcgov/hcap/blob/dev/server/services/employers.js#L130